### PR TITLE
Convert to context

### PR DESCRIPTION
**Pull request by syphernl**
## what
* Converted this module to use  `context.tf`

## why
* To follow the new standard applied across the other modules.

### CODE REVIEW

**Comment by syphernl** [comment]
I would really appreciate it if this PR could be tagged with `hacktoberfest-accepted` so it counts towards my [Hacktoberfest](https://hacktoberfest.digitalocean.com/) PR's (since the repo isn't tagged with `hacktoberfest`)

File: context.tf [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+# for example `context = module.this.context`,
+# and access individual variables as `module.this.<var>`,
+# with final values filled in.
+#
+# For example, when using defaults, `module.this.context.delimiter`
+# will be null, and `module.this.delimiter` will be `-` (hyphen).
+#
+
+module "this" {
+  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
```
Comment (joe-niland) [inline context.tf:22]: Need latest context.tf

File: main.tf [hunk 2]
```diff
@@ -1,12 +1,6 @@
 module "label" {
-  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
-  namespace   = var.namespace
-  stage       = var.stage
-  environment = var.environment
-  name        = var.name
-  attributes  = var.attributes
-  delimiter   = var.delimiter
-  tags        = var.tags
+  source  = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
```
Comment (joe-niland) [inline main.tf:2]: You could remove the label module

**Review by joe-niland — CHANGES_REQUESTED** [review]

File: context.tf (hunk 1 above)
Comment (syphernl) [reply context.tf]: Updated!

File: main.tf (hunk 2 above)
Comment (syphernl) [reply main.tf]: Removed and replaced.

**Review by joe-niland — APPROVED** [review]

**Pull request merged**